docs/guides/event_loop.md

```diff
@@ -2,14 +2,76 @@
 
 # An Introduction to the Event Loop in napari
 
+
+## tldr;
```

Comment:
Can we have a less casual heading here? Maybe "Brief summary" or something else.